### DISCUSSION TITLE
美国人权报告

### DISCUSSION THREAD

**Discussion by lxn7022** [The-Run-Philosophy-Organization/run#285] [General]
[https://baijiahao.baidu.com/s?id=1725990688912379722&wfr=spider&for=pc](url)

2021年，劣迹斑斑的美国人权状况进一步恶化。政治操弄导致新冠肺炎死亡病例激增，枪击事件致死人数再创新高，虚假民主践踏民众政治权利，暴力执法让移民难民的处境更加艰难，针对少数族裔特别是亚裔的歧视攻击愈演愈烈。与此同时，美国单边主义行径在全球制造了新的人道灾难。

——美国新冠肺炎确诊病例达3451万例，死亡病例达48万例，远超2020年，两项数据均居世界各国之首。人均预期寿命减少1.13岁，是自二战以来的最大降幅。

——美国社会治安形势恶化，暴力犯罪居高不下。全年共发生693起大规模枪击事件，比2020年增长10.1%。枪击事件导致超过4.4万人丧生。

——美国有49个州提出超过420项限制选民投票的法案，只有7%的美国年轻人认为美国民主制度尚属“健康”，民众对政府的信任度接近1958年以来的历史低点。

——81%的亚裔成年人认为针对亚裔群体的暴力行为正在增多。纽约市针对亚裔的仇恨犯罪比2020年猛增361%。59%的美国人认为少数族裔难以拥有平等的就业机会。

——2021财年，美国在南部边境拘留了170多万移民，其中包括4.5万名儿童。暴力执法夺走557人生命，比上一财年增长一倍多，为1998年以来最高值。

——美军撤离阿富汗时发动的空袭，造成一户阿富汗家庭10人被炸死，其中包括7名儿童，最小的年仅两岁。美国在关塔那摩监狱仍关押着39人。

**Comment by ghost** [comment]
https://zh.amnesty.org/content-type/2021-annual-report-usa/

看什么百家号的人权报告......

**Reply by SunsetMkt** [reply]
应该是这个吧：http://www.xinhuanet.com/2022-02/28/c_1128421287.htm

**Reply by ghost** [reply]
> 应该是这个吧：http://www.xinhuanet.com/2022-02/28/c_1128421287.htm

这个国际组织算歪屁股吗？基本上哪个国家都有......
[美国是全球最大的人权赤字国](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%86%85%E5%AE%A3/%E7%BF%BB%E8%BD%A6#%E7%BE%8E%E5%9B%BD%E6%98%AF%E5%85%A8%E7%90%83%E6%9C%80%E5%A4%A7%E7%9A%84%E4%BA%BA%E6%9D%83%E8%B5%A4%E5%AD%97%E5%9B%BD=) --- call me by your name

**Reply by SunsetMkt** [reply]
> > 应该是这个吧：http://www.xinhuanet.com/2022-02/28/c_1128421287.htm
> 
> 这个国际组织算歪屁股吗？基本上哪个国家都有...... [美国是全球最大的人权赤字国](https://github.com/The-Run-Philosophy-Organization/run/tree/main/%E5%86%85%E5%AE%A3/%E7%BF%BB%E8%BD%A6#%E7%BE%8E%E5%9B%BD%E6%98%AF%E5%85%A8%E7%90%83%E6%9C%80%E5%A4%A7%E7%9A%84%E4%BA%BA%E6%9D%83%E8%B5%A4%E5%AD%97%E5%9B%BD=) --- call me by your name

不知道欸...
不过根据摘录内容，楼主（？）要的应该是新华社那个。
有点off-topic了？

**Comment by lxn7022** [comment]
[http://www.xinhuanet.com/2022-02/28/c_1128421287.htm](url)
![image](https://user-images.githubusercontent.com/66630801/165885134-56dca405-0b77-4660-b03f-ef14835c416a.png)
[http://www.xinhuanet.com/2022-02/28/c_1128421287.htm](url)

**Reply by ghost** [reply]
我们是来交流的，而不是看你重复发同样的内容

**Comment by Hu-Wentao** [comment]
2年过去了，现在的看法如何呢？
